.pipelines/e2e-with-billing-all-regions.yml

```diff
@@ -0,0 +1,94 @@
+# Azure DevOps Pipeline running RP e2e and Billing e2e in all supported regions
+variables:
+  - template: vars.yml
+stages:
+  - stage: Sign_Off_Stage
+    displayName: Sign Off
+    dependsOn: []
+    jobs:
+    - job: Sign_off_job
+      pool:
+        name: ARO-E2E
+      steps:
+      - script: echo "Kick off E2E"
+  - template: ./templates/template-rp-and-billing-e2e-section.yml
+    parameters:
+      last_stage_of_previous_section: Sign_Off_Stage
+      e2e_subscription: $(e2e-subscription)
+      aro_v4_e2e_devops_spn: $(aro-v4-e2e-devops-spn)
+      billingE2EPipelineName: $(BillingE2EPipelineName)
+      billingE2EBranchName: $(BillingE2EBranchName)
+      gobin: ${{ variables.GOBIN }}
+      gopath: ${{ variables.GOPATH }}
+      goroot: ${{ variables.GOROOT }}
+      modulePath: ${{ variables.modulePath }}
+      locations:
+      - australiaeast
+      - southeastasia
+  - stage: Low_traffic_section_Summary
+    displayName: Low Traffic Section Summary
+    dependsOn:
+      - australiaeast_Billing_E2E
+      - southeastasia_Billing_E2E
+    jobs:
+    - job: Summary
+      pool:
+        name: ARO-E2E
+      steps:
+      - script: echo "Low_traffic_section_Summary -- TODO -- Send notification?."
+  - template: ./templates/template-rp-and-billing-e2e-section.yml
+    parameters:
+      last_stage_of_previous_section: Low_traffic_section_Summary
+      e2e_subscription: $(e2e-subscription)
+      aro_v4_e2e_devops_spn: $(aro-v4-e2e-devops-spn)
+      billingE2EPipelineName: $(BillingE2EPipelineName)
+      billingE2EBranchName: $(BillingE2EBranchName)
+      gobin: ${{ variables.GOBIN }}
+      gopath: ${{ variables.GOPATH }}
+      goroot: ${{ variables.GOROOT }}
+      modulePath: ${{ variables.modulePath }}
+      locations:
+      - westeurope
+      - northeurope
+      - uksouth
+  - stage: Europe_Section_Summary
+    displayName: Europe Section Summary
+    dependsOn:
+      - westeurope_Billing_E2E
+      - northeurope_Billing_E2E
+      - uksouth_Billing_E2E
+    jobs:
+    - job: Summary
+      pool:
+        name: ARO-E2E
+      steps:
+      - script: echo "Europe_Section_Summary -- TODO -- Send notification?."
```

Comment:
are these summary stages used for anything right now?

Reply:
Not yet, currently them are used to make the boundaries of each sector more straight forward on UI. And probably add email notification or ICM alert later on. 
https://msazure.visualstudio.com/AzureRedHatOpenShift/_build/results?buildId=30692967&view=results